### PR TITLE
ADEN-7508 Expose some methods for AE3Prebid integration

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-bidders/prebid/price-helper.js [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
 
+const DEFAULT_MAX_CPM = 20;
+const videoBiddersCap50 = ['appnexusAst', 'rubicon', 'wikiaVideo']; // bidders with $50 cap
+
 function isValidPrice(bid) {
 	return bid.getStatusCode && bid.getStatusCode() === Prebid.validResponseStatusCode;
 }
 
-export const DEFAULT_MAX_CPM = 20;
+export function transformPriceFromCpm(cpm, maxCpm) {
```
[the remark this answers is not in the document]
Comment (fraszczakszymon) [reply src/ad-bidders/prebid/price-helper.js]: L12 not needed then

File: src/ad-products/templates/uap/themes/hivi/hivi-theme.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
+import { utils } from '@wikia/ad-engine';
 import AdvertisementLabel from '../../../interface/advertisement-label';
 import { BigFancyAdTheme } from '../theme';
 import CloseButton from '../../../interface/close-button';
 import { Stickiness } from './stickiness';
 
+const logGroup = 'HiviTheme';
```
Comment (fraszczakszymon) [inline src/ad-products/templates/uap/themes/hivi/hivi-theme.js]:
```suggestion
const logGroup = 'hivi-theme';
```

please

File: src/ad-bidders/prebid/price-helper.js [hunk 3]
```diff
@@ -1,11 +1,38 @@
...
+	maxCpm = maxCpm || DEFAULT_MAX_CPM;
+
+	if (maxCpm < DEFAULT_MAX_CPM) {
```
Comment (fraszczakszymon) [inline src/ad-bidders/prebid/price-helper.js]:
```suggestion
	Math.max(maxCpm, DEFAULT_MAX_CPM);
```

File: src/ad-bidders/prebid/price-helper.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmnsobczak) [reply src/ad-bidders/prebid/price-helper.js]: Applied.

File: src/ad-products/templates/uap/themes/hivi/hivi-theme.js (hunk 2 above)
Comment (dmnsobczak) [reply src/ad-products/templates/uap/themes/hivi/hivi-theme.js]: After merge with dev it's not needed anymore.

File: src/ad-bidders/prebid/price-helper.js (hunk 3 above)
Comment (dmnsobczak) [reply src/ad-bidders/prebid/price-helper.js]: Applied.

**Pull request merged**